src/Components/App/App.module.css

```diff
@@ -1,8 +1,23 @@
-.wrap {
-  margin: auto;
-  width: 350px;
+body {
+  font-family: Helvetica, sans-serif;
 }
 
-.title {
-  font-family: Helvetica, sans-serif;
+.link {
+  text-decoration: none;
+  color: black;
+}
+
+.sidebar {
+  grid-area: sidebar;
 }
+
+.content {
+  grid-area: content;
+}
+
+.wrap {
+  display: grid;
+  grid-template-areas: 'sidebar ... content content' ;
+  margin: auto;
+  width: 500px;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Olga_project_React/pullRequest?prid=6401172)